src/main/java/fr/uge/lp/databackbonepoc/controllers/response/AlgorithmResponse.java

```diff
@@ -21,6 +23,20 @@ public enum AlgorithmType {
     MATCHING
   }
 
+  public record AlgorithmResponseWithValues(String name, Map<AlgorithmParameterResponse, String> parameters) {
+    public static AlgorithmResponseWithValues fromAlgorithmModel(fr.uge.lp.databackbonepoc.models.local.Algorithm algorithm) {
```

Comment:
```suggestion
    public static AlgorithmResponseWithValues fromAlgorithmModel(fr.uge.lp.databackbonepoc.models.local.Algorithm algorithm) {
```

Pourquoi pas le mettre en import ?